masterlist.yaml

```diff
@@ -17332,6 +17332,29 @@ plugins:
       - crc: 0x114081E6
         util: 'SSEEdit v4.0.4b'
 
+  - name: 'DeadMansRefuge.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194'
+        name: 'Dead Man''s Refuge (Nexus Mods)'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283320'
+        name: 'Dead Man''s Refuge (Bethesda.net)'
+    msg:
+      - <<: *patchIncluded
+        subs: [ '[Arena - An Encounter Zone Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/33487)' ]
+        condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
+    clean:
+      - crc: 0x683B8666
+        util: 'SSEEdit v4.0.4'
+  - name: 'DeadMansRefugeArenaPatch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194'
+        name: 'Dead Man''s Refuge - Arena Patch (Nexus Mods)'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283327'
+        name: 'Dead Man''s Refuge - Arena Patch (Bethesda.net)'
+    clean:
+      - crc: 0x43F3A723
+        util: 'SSEEdit v4.0.4'
```

Comment:
Okay, just needs a few format changes for the locations.

```suggestion
  - name: 'DeadMansRefuge(ArenaPatch)?\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194/'
        name: 'Dead Man''s Refuge (Nexus Mods)'
  - name: 'DeadMansRefuge.esp'
    url:
      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283320/'
        name: 'Dead Man''s Refuge (Bethesda.net)'
    msg:
      - <<: *patchIncluded
        subs: [ '[Arena - An Encounter Zone Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/33487)' ]
        condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
    clean:
      - crc: 0x683B8666
        util: 'SSEEdit v4.0.4'
  - name: 'DeadMansRefugeArenaPatch.esp'
    url:
      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283327/'
        name: 'Dead Man''s Refuge - Arena Patch (Bethesda.net)'
    clean:
      - crc: 0x43F3A723
        util: 'SSEEdit v4.0.4'
```

Reply:
You can use the commit suggestion button to apply changes.

![image](https://user-images.githubusercontent.com/1944639/183497374-495a9992-4159-4159-adc8-dbf0e98e22d0.png)